include/boost/json/detail/object.hpp

```diff
@@ -73,6 +75,19 @@ class unchecked_object
     }
 };
 
+template<class CharRange>
+std::pair<key_value_pair*, std::size_t>
+find_in_object(
+    object const& obj,
+    CharRange key) noexcept;
+
+extern template
+BOOST_JSON_DECL
+std::pair<key_value_pair*, std::size_t>
+find_in_object<string_view>(
+    object const&,
+    string_view key) noexcept;
```

Comment:
The "extern template" thing is a tiny bit risky, in the sense that there is a higher chance it will fail on some compiler.  There is also the question of linkage and export, for shared libraries. It would be better if this could be expressed without explicit function template instantiation, even if it means forward declaring a type specific to JSON Pointer.